docs/readme.md

```diff
@@ -499,19 +499,19 @@ implicit def obsToCancelable(obs: Obs): Cancelable = {
 ```
 
 ```scala mdoc:js
-implicit val ctx: Ctx.Owner = Ctx.Owner.safe()
-val showMessage = Var(false)
-val component = div(
-  Rx{
-    if( showMessage() )
-        div("Hello scala.rx")
-    else
-        button("show", onClick(true) --> showMessage)
-  },
-  managed(() => showMessage.foreach(println))
-)
-
-OutWatch.renderReplace("#example", div("yooo"))
+// implicit val ctx: Ctx.Owner = Ctx.Owner.safe()
```

Comment:
It didn't compile because of a missing implicit. I actually wanted to test the mdoc fix.

I'd like to keep the scala.rx dependency to be able to provide examples using scala.rx.